gcn_email_service/__init__.py

```diff
@@ -100,7 +105,15 @@ def recieve_alerts(consumer):
 @on_exception(expo, RateLimitException)
 @limits(calls=MAX_SENDS, period=1)
 def send_raw_ses_message_to_recipient(client, message, recipient):
-    BODY_TEXT = str(email.message_from_bytes(message.value()))
+    BODY_TEXT = ""
+    file = tempfile.NamedTemporaryFile()
+    global ATTACHMENT
```

Comment:
Globals?!! Not globals!!

Reply:
I am going to rewrite this into a Class structure, which will hopefully remove that

Reply:
Actually jsut fixed the scope, will be in next commit